docs/design/design_v1.9.md

```diff
@@ -440,14 +490,21 @@ Please note that:
 - The temporary authentication resolve to a user member of  `system:bootstrappers:kubeadm:default-node-token` group which was granted access to CSR api during the `kubeadm init` process
 - The automatic CSR approval is managed by the csrapprover controller, according with configuration done the `kubeadm init` process
 
+### (optional) Write init kubelet configuration
+
+If kubeadm is invoked with `--feature-gates=DynamicKubeletConfig`: 
+
+1. Read the kubelet base configuration from the `kubelet-base-config-v1.9`  ConfigMap in the `kube-system` namespace, and write it to disk as kubelet init configuration file  `/var/lib/kubelet/config/init/kubelet`.
```

Comment:
`kubelet-base-config-v1.9` ConfigMap in the `kube-system` namespace using the Bootstrap Token credentials, and write it to ...
nit: one space instead of two spaces here

You could point out that this will make the kubelet start if the right params (also outlined above, but could be copied here as well) are set